vendor/brave-ios/Ads/BATBraveAds.mm

```diff
@@ -174,6 +175,17 @@ - (void)setEnabled:(BOOL)enabled
   }
 }
 
+- (BOOL)shouldShowPublisherAdsOnParticipatingSites
```

Comment:
This property matches the method `shouldShowPublisherAdsOnParticipatingSites` from the AdsClient so we'll have to change one of them or else this won't compile (dupe symbol)

Reply:
Fixed